### DISCUSSION TITLE
IMDB search help, only specified countries and language

### DISCUSSION THREAD

**Discussion by terryalex64** [meisnate12/Plex-Meta-Manager#573] [Q&A] [answered]
I am trying to make a search that only has movies from the following countries, United States US, Australia, AU.
I tried using: https://www.imdb.com/search/title/?title_type=feature,tv_movie&user_rating=7.0,10.0&num_votes=5000,&genres=action&countries=us,au

and :
I tried using: https://www.imdb.com/search/title/?title_type=feature,tv_movie&user_rating=7.0,10.0&num_votes=5000,&genres=action&countries=us&countries=au

Both of these give the same results, movies that were produced in both countries. Is there any way besides creating another country-specific search for each?

I also tried creating a search for English language movies but it returned a lot of movies that were in foreign languages but also had English listed in the movie details. Any syntax to use to specify no other languages?

**Comment by meisnate12** [comment, marked as answer]
the filter for `original_language` might be helpful also you could just do two searches

https://www.imdb.com/search/title/?title_type=feature,tv_movie&user_rating=7.0,10.0&num_votes=5000,&genres=action&countries=us
https://www.imdb.com/search/title/?title_type=feature,tv_movie&user_rating=7.0,10.0&num_votes=5000,&genres=action&countries=au

**Reply by terryalex64** [reply]
Thank you for the quick response and thank you for the great program.
